package.json

```diff
@@ -15,17 +15,18 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 -c .storybook -s src/static",
     "build-storybook": "build-storybook -s ./src/static",
-    "test": "npm run test-lint && prettier --check ./src/**/* && node ./scripts/build.test.js && npm run test-jest && npm run test-chrome",
-    "test-lint": "eslint src/",
+    "test": "npm run eslint && prettier --check ./src/**/* && node ./scripts/build.test.js && npm run test-jest && npm run test-chrome",
+    "eslint": "eslint src/",
     "test-chrome": "sh scripts/testcafe-chrome.sh",
     "test-ie": "sh scripts/testcafe-ie.sh",
     "test-jest": "jest --env=jsdom",
     "test-jest-coverage": "jest --env=jsdom --coverage",
     "test-jest-watch": "jest --env=jsdom --watch -o",
     "test-ui": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome ./**/ui.test.js",
-    "test-ui-hl": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
+    "test-ui-headless": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
     "new": "node scripts/create-a-m-o/index.js",
-    "deploy-branch": "npm run build-storybook && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment.sh && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment-notifier.sh"
+    "deploy-branch": "npm run build-storybook && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment.sh && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment-notifier.sh",
+    "test:e2e:browserstack": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 BROWSERSTACK_USERNAME=danielb17 BROWSERSTACK_ACCESS_KEY=z9ereHMoK6twktEx84wa testcafe 'browserstack:chrome:Windows' ./**/datepicker/ui.test.js"
```

Comment:
Never publish a key to a public repo (even non-public ones). You have to make this one invalid. There are scripts that automatically scan github, bitbucket, etc... for stuff like that.